gen.go

```diff
@@ -50,7 +50,9 @@ package carrow
 
 /*
 #cgo pkg-config: arrow plasma
-#cgo LDFLAGS: -lcarrow -L.
+#cgo LDFLAGS: -lcarrow
+#cgo linux LDFLAGS: -L./bindings/linux-x86_64
+#cgo darwin LDFLAGS: -L./bindings/darwin-x86_64
```

Comment:
I we don't have darwin bindings - let's delete this line until we have and add an issue

Reply:
#44